tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -0,0 +1,388 @@
+                                                    skipcleanup=True, setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 4: Performed WRITEs/READs/Verify with variable sizes objects.")
+        LOGGER.info("Step 5: Restore server pod and check cluster status.")
+        resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                       restore_method=self.restore_method,
+                                       restore_params={"deployment_name": self.deployment_name,
+                                                       "deployment_backup":
+                                                           self.deployment_backup},
+                                       clstr_status=True)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to restore server pod by {self.restore_method} "
+                                          "way OR the cluster is not online")
+        LOGGER.info("Step 5: Successfully started server pod and cluster is online.")
+        self.restore_pod = False
+        LOGGER.info("Step 6: Perform READs and verify DI on the written data in degraded mode")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix_deg,
+                                                    skipwrite=True, skipcleanup=True,
+                                                    setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 6: Successfully run READ/Verify on data written in degraded mode")
+        LOGGER.info("Step 7: Perform READ/Verify on data written in healthy mode")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipwrite=True,
+                                                    skipcleanup=True, setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 7: Successfully run READ/Verify on data written in healthy mode")
+        LOGGER.info("Step 8: Start IOs again after server pod restart by delete deployment")
+        self.test_prefix = 'test-44834-1'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipcleanup=True, setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 8: Successfully IOs completed after server pod restart by delete "
+                    "deployment")
+        LOGGER.info("ENDED: Test to verify READs after server pod restart.")
+
+    # pylint: disable=too-many-statements
```

Comment:
If you disable in 1st test, you won't need to disable the same in following tests.